src/window.rs

```diff
@@ -60,46 +59,45 @@ impl Window {
         };
 
         if self.frame_count == 0 {
+            // NOTE(mbernat): It's possible to avoid initial mode setting
+            // since we're keeping the previous mode: we could just call page_flip directly.
```

Comment:
> since we're keeping the previous mode

Do you say this because we're selecting the connector's preferred mode? I'm not sure where in the code we preserve the previous mode, otherwise.

Reply:
Yeah, exactly. It seems to be quite a common scenario for displays to use their preferred modes. So if the previous mode matches the next mode you don't need to do full mode setting, it's sufficient (and I think recommended) to just change the buffer with a page flip.

But it's really just a tiny optimization that I wrote down in passing.